src/pages/wallet-overview/wallet-overview.spec.ts

```diff
@@ -130,10 +145,25 @@ describe("WalletOverviewPage", () => {
     });
   })
 
-  it("should return false after deleting a wallet that doesn't exist", () => {
-    let result = comp.deleteSelectedWallet(null);
+  it("should open landing page and show a toast after deleting the only wallet", (done) => {
+    // Get all 3 wallets
+    comp.getAllWallets().then(data => {
+      // Delete first one
+      comp.deleteSelectedWallet(comp.currentWallet);
```

Comment:
The test should succeed when the app navigates to the landing page __only__ when the last wallet was deleted. The current test doesn't fully check this. If the app navigates back to the landing page when deleting the third wallet there is no check to catch this scenario.

Possible solution: after each call to `deleteSelectedWallet` check if the appropriate functions have __not__ been called. Then when the last wallet was deleted check if they __have__ been called.